app/settings/manifest.js

```diff
@@ -143,6 +143,77 @@ vote at ${issue(125,true)}.
             "type": "checkbox",
             "label": future_i18n('Sort open windows to the top, scroll to the top of the list'),
         },
+        {
+            "tab": future_i18n("Behaviour"),
+            "group": future_i18n("When I..."),
+            "name": S.FIXED_WINDOW_SIZE,
+            "type": "checkbox",
+            "label": future_i18n('Open a window, always open it with a preferred fixed size'),
+        },
+        {   // some extra descriptive text for S.FIXED_WINDOW_SIZE
+            "tab": future_i18n("Behaviour"),
+            "group": future_i18n("When I..."),
+            "type": "description",
+            "text": future_i18n(`If you change this value please refresh Settings
+                    page (this page) to reveal preferred window size parameter`)
```

Comment:
Could you give me a bit more detail about this?  We can update the settings page at load time (app/settings/settings.js. `main()`) if we need to populate a field or customize something.